src/commands/generate/delta.ts

```diff
@@ -479,46 +519,46 @@ export default class GenerateDelta extends BaseCommand<typeof GenerateDelta> {
           // Call the .toRuby verbose if the log level is debug or verbose
           const processLogLevel = Boolean(logLevel === 'debug' || logLevel === 'verbose')
           if (index >= 0) {
-          // We found a mapping for this control (aka index >=0)
-          // The new control (control) has the new metadata but doesn't have
-          // the describe block (code). Using the updateControl method with the new
-          // control so we can get the code with the new metadata.
-
+            // We found a mapping for this control (aka index >=0)
+            // The new control (control) has the new metadata but doesn't have
+            // the describe block (code). Using the updateControl method with the new
+            // control so we can get the code with the new metadata.
 
-            // TODO: Can use the getExistingDescribeFromControl(existingProfile.controls[index]) // skipcq: JS-0099
+            // NOTE: Can use the getExistingDescribeFromControl(existingProfile.controls[index])
             //       method from inspect-objects
-            const newControl = updateControl(existingProfile.controls[index], control, logger)
-
-            logger.debug(`Writing updated control with code block for: ${control.id}.`)
+            const newControl = updateControl(existingProfile.controls[index], control, thisLogger)
+            this.logThis(`Writing updated control with code block for: ${control.id}.`, 'info')
             fs.writeFileSync(path.join(outputProfileFolderPath, 'controls', `${control.id}.rb`), newControl.toRuby(processLogLevel))
           } else {
-          // We didn't find a mapping for this control - Old style of updating controls
-            logger.debug(`Writing new control without code block for: ${control.id}.`)
+            // We didn't find a mapping for this control - Old style of updating controls
+            this.logThis(`Writing new control without code block for: ${control.id}.`, 'info')
             fs.writeFileSync(path.join(outputProfileFolderPath, 'controls', `${control.id}.rb`), control.toRuby(processLogLevel))
           }
         })
 
-        logger.info(`  Writing delta file for ${existingProfile.title}`)
+        // logger.info(`  Writing delta file for ${existingProfile.title}`)
```

Comment:
I am fine with deleting these logger comments in favor of the new versions that you are using